mock/angular-mocks/router-mock.ts

```diff
@@ -3,15 +3,11 @@ export class RouterMock {
 
   routerState = { root: '' };
 
-  navigate = jasmine.createSpy('navigate')
-    .and
-    .returnValue(Promise.resolve(true));
+  navigate = jasmine.createSpy('navigate').and.returnValue(Promise.resolve(true));
```

Comment:
for these you can simply use resolveTo takes out the need for the returnValue(Promise.resolve(true)) - same for all instances where promise.resolve is used